### PR TITLE
llama-pack => Bubblewrap

### PR DESCRIPTION
**Pull request by Maxim-Mazurok**
What's changed, or what was fixed?
- Replaced "llama-pack" with "bubblewrap"
- Replaced screenshot "npx llama-pack" with "npx @cli/bubblewrap"
- Optimized it using `optipng -snip -strip all -o7 -zm1-9 bubblewrap-init-custom-start-url.png`

**Fixes:** https://github.com/GoogleChromeLabs/bubblewrap/issues/185, ref: https://github.com/google/WebFundamentals/pull/8552

**Target Live Date:** YYYY-MM-DD (should be live ASAP, I think)

- [x] This has been reviewed and approved by (@andreban)
- [x] I have run `npm test` locally and all tests pass.
- [x] I have added the appropriate `type-something` label.
- [x] I've staged the site and manually verified that my content displays correctly.

**CC:** @petele

---

Sorry, can't run tests on Windows cmd shell:
```
[18:39:00] ERROR: gulp-tasks/test.js A critical gulp task exception occurred: Command failed: git --no-pager diff --name-only $(git merge-base origin/master HEAD)
fatal: ambiguous argument '$(git': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'
```

### CODE REVIEW

**Comment by Maxim-Mazurok** [comment]
Ran tests:
```
[18:56:55] ---------------------------------
[18:56:55] WebFundamentals Gulp
[18:56:55] ---------------------------------
[18:56:55] Build RSS Files: false
[18:56:55] 
[18:56:55] Using gulpfile /private/tmp/WebFundamentals/gulpfile.js
[18:56:55] Starting 'test:travis-init'...
[18:56:55]   Not Travis.
[18:56:55] Finished 'test:travis-init' after 5.3 ms
[18:56:55] Starting 'test'...
[18:56:55]  Searching for files...
[18:56:55]   Searching for changed files
[18:56:55]   $ git --no-pager diff --name-only $(git merge-base origin/master HEAD)
[18:56:56]   $ git --no-pager diff --name-only $(git merge-base origin/master HEAD) OK
[18:56:56] Testing files...
[18:56:56] WARNING: src/content/en/updates/images/2020/03/llama-pack-init-custom-start-url.png Unable to read file stats: ENOENT: no such file or directory, stat 'src/content/en/updates/images/2020/03/llama-pack-init-custom-start-url.png'
[18:56:56] 
[18:56:56] Test Completed.
[18:56:56] Files tested:   5
[18:56:56]  - with issues: 1
[18:56:56] Errors  :  0
[18:56:56] Warnings:  1
[18:56:56] Finished 'test' after 390 ms
```

File: src/content/en/android/trusted-web-activity/query-parameters/index.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ The wizard will use the start_url from the Web Manifest as default and will ask
 the value, giving developers the change to add extra parameters to the url used to start the
 Progressive Web App.
 
-![Showing the Bubblewrap CLI output](/web/updates/images/2020/03/llama-pack-init-custom-start-url.png)
+![Showing the Bubblewrap CLI output](/web/updates/images/2020/03/bubblewrap-init-custom-start-url.png)
```
Comment (andreban) [inline src/content/en/android/trusted-web-activity/query-parameters/index.md:48]: I ran `npm test` locally. This image address is invalid, as the file was renamed to `src/content/en/updates/images/2020/03/bubblewrap-pack-init-custom-start-url.png` - There's an extra `-pack` segment . We should probably rename the file to `src/content/en/updates/images/2020/03/bubblewrap-init-custom-start-url.png`

Comment (Maxim-Mazurok) [reply src/content/en/android/trusted-web-activity/query-parameters/index.md:48]: Good catch! I didn't catch that during `npm test`...
Fix: https://github.com/google/WebFundamentals/pull/8732/commits/7ca6e1049032040f9e8315a9d8d47bd0df506241

**Comment by andreban** [comment]
I have staged locally and the content looks good. Thank you for the PR.

**Comment by Maxim-Mazurok** [comment]
I also did, just to make sure :) Thanks for bubblewrap!

**Comment by petele** [comment]
Thanks for the PR! If you delete or rename a file, be sure to set up a redirect so if anyone links to the old content, they're seamlessly redirected to the renamed file. I'll fix in a separate commit.

**Pull request merged**